test/parallel/test-fs-glob.mjs

```diff
@@ -1,12 +1,18 @@
 import * as common from '../common/index.mjs';
 import tmpdir from '../common/tmpdir.js';
-import { resolve, dirname, sep } from 'node:path';
+import { resolve, dirname, sep, basename } from 'node:path';
 import { mkdir, writeFile, symlink, glob as asyncGlob } from 'node:fs/promises';
-import { glob, globSync } from 'node:fs';
+import { glob, globSync, Dirent } from 'node:fs';
 import { test, describe } from 'node:test';
 import { promisify } from 'node:util';
 import assert from 'node:assert';
 
+function assertDirents(dirents) {
+  dirents.forEach((dirent) => {
+    assert(dirent instanceof Dirent);
+  });
```

Comment:
```suggestion
  assert.ok(dirents.every((dirent) => dirent instanceof Dirent));
```

Reply:
nit: Wouldn't that be harder to debug since you won't know which direndt failed

Reply:
In both cases, you wouldn't know, would you?

Reply:
Yes sorry, that was more of a comment to both options (vs. for example defining a `t.test` subtest for each)